ppr-api/src/ppr_api/models/search_request.py

```diff
@@ -114,7 +114,7 @@ def update_search_selection(self, search_json):
     def search_by_registration_number(self):
         """Execute a search by registration number query."""
         reg_num = self.request_json['criteria']['value']
-        result = db.session.execute(search_utils.REG_NUM_QUERY, {'query_value': reg_num})
+        result = db.session.execute(search_utils.REG_NUM_QUERY, {'query_value': reg_num.strip().upper()})
```

Comment:
might want to safe guard this in case reg_num is null (unless validation protects this)

Reply:
You are right, search criteria null/empty string fails validation so it is already covered.